e2e/keywords/volume.resource

```diff
@@ -37,6 +37,12 @@ Create volume ${volume_id} from ${workload_kind} ${workload_id} volume backup ${
     ${backup_url}=    get_backup_url    ${backup_id}    ${workload_volume_name}
     create_volume   ${volume_name}    size=3Gi    numberOfReplicas=3    fromBackup=${backup_url}
 
+Create volume ${target_volume_id} from volume ${source_volume_id} backup ${backup_id}
+    ${target_volume_name}=    generate_name_with_suffix    volume    ${target_volume_id}
+    ${source_volume_name}=    generate_name_with_suffix    volume    ${source_volume_id}
+    ${backup_url}=    get_backup_url    ${backup_id}    ${source_volume_name}
+    create_volume   ${target_volume_name}    numberOfReplicas=3    fromBackup=${backup_url}
+
```

Comment:
Can the keyword `Create volume ${target_volume_id} from volume ${source_volume_id} backup ${backup_id}` be reused in the code at https://github.com/longhorn/longhorn-tests/blob/e5e585a0a1d461554c037b3d0d7a1b5e7d1b8b31/e2e/keywords/volume.resource#L26-L31 ? I think defining `${target_volume_id}` is better than `${volume_id}`.

Reply:
Yes, I replaced it with the `Create volume ${target_volume_id} from backup ${backup_id} of volume ${source_volume_id}` keyword. Thank you!